extensions/3rdparty/tabber/tabber.php

```diff
@@ -21,35 +29,23 @@ function wfTabber( Parser $parser ) {
 	return true;
 }
 
-function renderTabber( $paramstring, $params, $parser ) {
-	global $wgExtensionsPath, $wgStyleVersion;
-
-	$path = $wgExtensionsPath . '/3rdparty/tabber/';
-
-	/*
-	 * Wikia Change Start @author: marzjan
-	 */
-	$snippets = JSSnippets::addToStack(
-		array( '/extensions/3rdparty/tabber/tabber.js' ),
-		array( '$.loadJQueryUI' )
-	);
+function renderTabber( $paramstring, $params, Parser $parser ) {
+	// Wikia change - add styles and scripts to output:
+	$parser->getOutput()->addModules( 'ext.tabber' );
 
-	$htmlHeader = '<link rel="stylesheet" href="' . $path . 'tabber.css?' . $wgStyleVersion . '" TYPE="text/css" MEDIA="screen">'
-		. $snippets
-		. '<div class="tabber">';
-	$htmlFooter = '</div>';
-	/*
-	 * Wikia Change End
-	 */
-
-	$htmlTabs = "";
+	// Wikia change - hide tabbers by default and let JS show them -> for lazy loading images
+	$style = 'display:none;';
 
 	$arr = explode( "|-|", $paramstring );
+	$htmlTabs = Html::openElement( 'div', [ 'class' => 'tabber', 'style' => $style ] );
+
 	foreach ( $arr as $tab ) {
 		$htmlTabs .= buildTab( $tab, $parser ); # macbre: pass Parser object (refs RT #34513)
 	}
 
-	return $htmlHeader . $htmlTabs . $htmlFooter;
+	$htmlTabs .= Html::closeElement( 'div' );
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "$htmlTabs". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)